crates/wasmcloud-provider/Cargo.toml

```diff
@@ -22,23 +22,30 @@ rustls-tls = ["kube/rustls-tls", "kubelet/rustls-tls", "krator/rustls-tls"]
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-wascc-host = "0.13"
+# Ideally, we should plumb the wasm3 vs wasmtime feature up into the top level build so people can
+# choose their own, but right now the wasm3 stuff doesn't work with windows without a clang
+# dependency
+# wasmcloud-host = { version = "0.16", default-features = false, features = ["wasmtime"] }
+wasmcloud-host = { git = "https://github.com/wasmCloud/wasmCloud.git", version = "0.16", branch = "main", default-features = false, features = ["wasmtime"] }
```

Comment:
note to not merge until the new wasmcloud-host version is out.

Reply:
new wasmcloud-host version is out as of now 😄 try `0.16.1`